src/fontra/client/core/utils.js

```diff
@@ -232,7 +234,20 @@ export async function writeToClipboard(clipboardObject) {
     });
   }
 
-  navigator.clipboard.write([new ClipboardItem(clipboardItemObject)]);
+  try {
+    await navigator.clipboard.write([new ClipboardItem(clipboardItemObject)]);
+  } catch {
+    // Write at least the plain/text MIME type to the clipboard
+    if (clipboardObject["text/plain"])
+      try {
```

Comment:
I'm not sure this second try/catch is needed.

Reply:
Safari throws an error for unhandled promise 
This can be ` await navigator.clipboard.writeText(clipboardObject["text/plain"].catch(error => {})` but the need remains

Reply:
You're saying that a plain
```js
await navigator.clipboard.write([new ClipboardItem(clipboardItemObject)])
```
without try/catch doesn't work here?

Reply:
if you do it without `await`, It works but Safari says this:
`Unhandled Promise Rejection: NotAllowedError: The request is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.`

If I write this: 
```
if (clipboardObject["text/plain"])
      await navigator.clipboard.writeText(clipboardObject["text/plain"]);

writeToLocalStorage(clipboardObject);
```
no errors, but `writeToLocalStorage()` is not executed...